sphinx/transforms/post_transforms/images.py

```diff
@@ -231,10 +228,7 @@ def guess_mimetypes(self, node: nodes.image) -> List[str]:
     def handle(self, node: nodes.image) -> None:
         _from, _to = self.get_conversion_rule(node)
 
-        if _from in node['candidates']:
-            srcpath = node['candidates'][_from]
-        else:
-            srcpath = node['candidates']['*']
+        srcpath = node['candidates'].get(_from, node['candidates']['*'])
```

Comment:
This one feels less clean than the others, but I'm not sure of any better refactoring.

A